peekingduck/pipeline/nodes/input/live.py

```diff
@@ -79,6 +79,9 @@ def __init__(self, config: Dict[str, Any]=None, **kwargs: Any) -> None:
             self.logger.info('Resizing of input set to %s by %s',
                              self.resize['width'],
                              self.resize['height'])
+        if not any(ext in config['filename'] for ext in ['.mov', '.avi', '.mp4']):
+            raise ValueError("filename extension must be one of: .avi, .mov, .mp4")
```

Comment:
1. It might be simpler to use string.split('.') and get the extension from the last element in the resulting list
2. Better to have 1 source of truth for the allowable types rather than update twice.
```
# this is from input.live
self._allowed_extensions = ["jpg", "jpeg", "png", "mp4", "avi", "mov", "mkv"]
```

Reply:
Updated